settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
Sorry, I spaced out and didn't read the second half of your paragraph. I should probably sleep soon!

I really want the names to line up if I can have it. If i can't fine.

What about putting it as a subdirectory of `docs`?